v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/v2g_globals.py

```diff
@@ -872,28 +874,7 @@ async def __set_select_options(self, entity_id: str, options: list, option_to_se
     #                METHODS FOR IO of SETTINGS FILE                     #
     ######################################################################
 
-    async def __retrieve_settings(self):
-        """Retrieve all settings from the settings file
-        """
-        self.log(f"__retrieve_settings called")
-
-        if not os.path.exists(self.settings_file_path):
-            self.log(f"__retrieve_settings, no settings file found: creating")
-            self.v2g_settings = {}
-            self.__write_to_file(self.v2g_settings)
```

Comment:
I left out this line from the refactored implementation: I do not think this is needed.
It only makes sense to write the file if there is actual content to write.